external/storm-eventhubs/src/main/java/org/apache/storm/eventhubs/spout/ZookeeperStateStore.java

```diff
@@ -60,7 +61,7 @@ public void close() {
     @Override
     public void saveData(String statePath, String data) {
         data = data == null ? "" : data;
-        byte[] bytes = data.getBytes();
+        byte[] bytes = DFSUtil.string2Bytes(data);
```

Comment:
I prefer to not introduce hadoop dependencies just for this single function since its implementation is really simple 
https://github.com/apache/hadoop/blob/trunk/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/DFSUtil.java#L273
https://github.com/apache/hadoop/blob/trunk/hadoop-hdfs-project/hadoop-hdfs-client/src/main/java/org/apache/hadoop/hdfs/DFSUtilClient.java#L114-L120

This `DFSUtil.string2Bytes(data)` method  is basically equivalent to `data.getBytes(StandardCharsets.UTF_8.name());`. 

Can you explain more about the issues you have encountered and maybe provide a test case? 

Thanks